components/automate-backend-elasticsidecar/habitat/bin/elastic_sidecar.rb

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env ruby
+
+require 'bcrypt'
+require 'fileutils'
+require 'http'
+require 'json'
+require 'logger'
+require 'toml-rb'
+require 'yaml'
+require 'mixlib/shellout'
+require "chefstyle"
```

Comment:
?

Reply:
So how we can use this chef-style , I was trying to use this for chef-style lintment